adafruit_lsm303.py

```diff
@@ -122,12 +122,32 @@
 # Magentometer rates
 MAGRATE_0_7                = const(0x00)  # 0.75 Hz
 MAGRATE_1_5                = const(0x01)  # 1.5 Hz
-MAGRATE_3_0                = const(0x62)  # 3.0 Hz
+MAGRATE_3_0                = const(0x02)  # 3.0 Hz
 MAGRATE_7_5                = const(0x03)  # 7.5 Hz
 MAGRATE_15                 = const(0x04)  # 15 Hz
 MAGRATE_30                 = const(0x05)  # 30 Hz
 MAGRATE_75                 = const(0x06)  # 75 Hz
-MAGRATE_220                = const(0x07)  # 200 Hz
+MAGRATE_220                = const(0x07)  # 220 Hz
+
+# Accelerometer gains
```

Comment:
This is not referred to as gain in any of our other drivers, or in the data sheet. The datasheet refers to it as "full-scales". However, we consistently refer to it as `range` in our other accelerometer drivers. Please change this to `# Accelerometer ranges` and change the variable names to match, e.g. `ACCEL_RANGE_2 =`, etc.